ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -274,6 +274,17 @@ interface {{ ethernet_interface.name }}
 {%             if ethernet_interface.dot1x.reauthorization_request_limit is arista.avd.defined %}
    dot1x reauthorization request limit {{ ethernet_interface.dot1x.reauthorization_request_limit }}
 {%             endif %}
+{%             if ethernet_interface.dot1x.eapol is arista.avd.defined %}
+{%                 if ethernet_interface.dot1x.eapol.disabled is arista.avd.defined (true) %}
+   dot1x eapol disabled
+{%                 elif ethernet_interface.dot1x.eapol.authentication_failure_fallback_mba is arista.avd.defined %}
+{%                     set auth_failure_fallback_mba = "dot1x eapol authentication failure fallback mba" %}
+{%                     if ethernet_interface.dot1x.eapol.authentication_failure_fallback_mba.enabled is arista.avd.defined(true) %}
```

Comment:
Looking at this logic again, I am not quite sure what is expected here

when you have this:

```
      eapol:
        disabled: false
        authentication_failure_fallback_mba:
          enabled: false
```

What do you expect to be rendered? It look to me that the check for 
`ethernet_interface.dot1x.eapol.authentication_failure_fallback_mba.enabled is arista.avd.defined(true)`

should be done line 280 instead

Reply:
Yes it correct. I will update it as you suggest. In my original code was:
if ethernet_interface.dot1x.eapol.authentication_failure_fallback_mba is arista.avd.defined and ethernet_interface.dot1x.eapol.disabled is arista.avd.defined(false)